src/instance/minecraft/commands/KuudraCommand.ts

```diff
@@ -0,0 +1,44 @@
+import { ChatCommandContext, ChatCommandHandler } from '../common/ChatInterface'
+import { Client, HypixelSkyblockMemberRaw } from 'hypixel-api-reborn'
+
+enum Kuudra {
+  'basic' = 'none',
+  'hot' = 'hot',
+  'burning' = 'burning',
+  'fiery' = 'fiery',
+  'infernal' = 'infernal'
+}
+
+export default {
+  triggers: ['kuudra', 'k'],
+  enabled: true,
+  handler: async function (context: ChatCommandContext): Promise<string> {
+    const givenUsername = context.args[1] ?? context.username
+    const tier = context.args[0] as keyof typeof Kuudra
```

Comment:
Switch those two lines to start args checking from 0.

Reply:
done